GiniVision/Classes/Core/GiniScreenAPICoordinator+Review.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 extension GiniScreenAPICoordinator: ReviewViewControllerDelegate {
     
     func review(_ viewController: ReviewViewController, didReview document: GiniVisionDocument) {
-        self.updateValueInDocuments(for: document)
+        updateValue(for: document)
```

Comment:
A name such as `updateValue` doesn't give me much information about what the method does.

Reply:
Since I will rename the `DocumentRequest`, it will make more sense to call it `updateDocument`